### PR TITLE
tomato changes

### PR DESCRIPTION
**Pull request by kanchan555-ui**
added acceptance test for tomato icon

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -13,6 +13,7 @@ object Dependencies {
     "com.typesafe"         % "config"                  % "1.3.2",
     "com.typesafe.play"   %% "play-ws-standalone-json" % "2.1.2",
     "com.typesafe.play"   %% "play-ahc-ws-standalone"  % "2.1.2",
+    "org.mongodb.scala"   %% "mongo-scala-driver"      % "2.6.0",
```
Comment (yangwong9664) [inline project/Dependencies.scala:16]: can this be removed now?

File: src/test/resources/features.SCAWrapper/local/SCAWrapperFrontPageLocal.feature [hunk 2]
```diff
@@ -1,6 +1,10 @@
 @suite
 Feature: Wrapper integrating with sca-frontend
 
+  Background:
+   # Given mongoDB is dropped
```
Comment (yangwong9664) [inline src/test/resources/features.SCAWrapper/local/SCAWrapperFrontPageLocal.feature:5]: remove?

File: src/test/resources/features.SCAWrapper/local/SCAWrapperFrontPageLocal.feature [hunk 3]
```diff
@@ -1,6 +1,10 @@
...
+    And A message is updated in reactive mongo
```
Comment (yangwong9664) [inline src/test/resources/features.SCAWrapper/local/SCAWrapperFrontPageLocal.feature:6]: Can this be updated to something like "And a message is posted to the messages API"?
Also And -> Given?

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/SCAWrapperStartPageSteps.scala [hunk 4]
```diff
@@ -224,5 +228,81 @@ class SCAWrapperStartPageSteps extends ScalaDsl with EN with Matchers with WebBr
...
   }
 
+    Then("""^mongoDB is dropped$""") { () =>
+    MongoConnection.dropDatabase("pertax-frontend")
+    MongoConnection.dropDatabase("enrolment-store-stub")
+    MongoConnection.dropDatabase("message")
+    MongoConnection.dropDatabase("pay-api")
+    MongoConnection.dropDatabase("citizen-details")
+    MongoConnection.dropDatabase("keystore")
+  }
```
Comment (yangwong9664) [inline src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/SCAWrapperStartPageSteps.scala:238]: Remove?

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/SCAWrapperStartPageSteps.scala [hunk 5]
```diff
@@ -224,5 +228,81 @@ class SCAWrapperStartPageSteps extends ScalaDsl with EN with Matchers with WebBr
...
+
+
+  Given("""A message is updated in reactive mongo""") {
+
+   val id=  Random.alphanumeric.filter(_.isDigit).take(14).mkString
+    val subject = Random.alphanumeric.filter(_.isLetter).take(4).mkString
+    //val id = Random.nextString(14)
+    //val subject = Random.nextString(50)
```
Comment (yangwong9664) [inline src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/SCAWrapperStartPageSteps.scala:246]: Remove?

File: src/test/scala/uk/gov/hmrc/test/ui/utils/MongoConnection.scala [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+package uk.gov.hmrc.test.ui.utils
+
+import org.mongodb.scala.MongoClient
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+object MongoConnection {
```
Comment (yangwong9664) [inline src/test/scala/uk/gov/hmrc/test/ui/utils/MongoConnection.scala:25]: Whole file remove?

**Review by yangwong9664 — CHANGES_REQUESTED** [review]

File: src/test/resources/features.SCAWrapper/local/SCAWrapperFrontPageLocal.feature (hunk 2 above)
Comment (kanchan555-ui) [reply src/test/resources/features.SCAWrapper/local/SCAWrapperFrontPageLocal.feature]: removed

File: src/test/resources/features.SCAWrapper/local/SCAWrapperFrontPageLocal.feature (hunk 3 above)
Comment (kanchan555-ui) [reply src/test/resources/features.SCAWrapper/local/SCAWrapperFrontPageLocal.feature]: this cant be done i have tried this but multiple message got publish which is equal to number of scenario hence I have removed this and added only to the message scenario

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/SCAWrapperStartPageSteps.scala (hunk 5 above)
Comment (kanchan555-ui) [reply src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/SCAWrapperStartPageSteps.scala]: removed

File: src/test/scala/uk/gov/hmrc/test/ui/utils/MongoConnection.scala (hunk 6 above)
Comment (kanchan555-ui) [reply src/test/scala/uk/gov/hmrc/test/ui/utils/MongoConnection.scala]: removed

**Review by yangwong9664 — DISMISSED** [review]

**Review by yangwong9664 — APPROVED** [review]

**Pull request merged**